ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/ImageCvePage.tsx

```diff
@@ -134,9 +161,31 @@ function ImageCvePage() {
         skip: entityTab !== 'Image',
     });
 
+    const deploymentDataRequest = useQuery<
+        { deploymentCount: number; deployments: DeploymentForCve[] },
+        {
+            query: string;
+            pagination: PaginationParam;
+        }
+    >(imageCveAffectedDeploymentsQuery, {
+        variables: {
+            query: getRequestQueryStringForSearchFilter({
+                ...querySearchFilter,
+                CVE: cveId,
+            }),
+            pagination: {
+                offset: (page - 1) * perPage,
+                limit: perPage,
+                sortOption,
+            },
+        },
+        skip: entityTab !== 'Deployment',
```

Comment:
`skip` will only run the query if the value is `false`. This makes it so that both the image table and deployment table queries can be part of the top level component without running extra network calls for the table that isn't currently being displayed. (Since we can't conditionally call the hooks.)

This felt cleaner than splitting the page into two components that had around 90% identical code.